drivers/gpio/gpio_bd8lb600fs.c

```diff
@@ -1,111 +1,50 @@
 /*
- * Copyright (c) 2022 SILA Embedded Solutions GmbH
+ * Copyright (c) 2024 SILA Embedded Solutions GmbH
```

Comment:
Add the new year without removing the previous one

Reply:
done